core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/instancegroup/InstanceGroupV4RequestToInstanceGroupConverter.java

```diff
@@ -1,26 +1,32 @@
 package com.sequenceiq.cloudbreak.converter.v4.stacks.instancegroup;
 
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-
-import javax.inject.Inject;
-
-import org.springframework.stereotype.Component;
-
-import com.sequenceiq.cloudbreak.common.mappable.ProviderParameterCalculator;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.instancegroup.InstanceGroupV4Request;
+import com.sequenceiq.cloudbreak.cloud.model.instance.AzureInstanceGroupParameters;
+import com.sequenceiq.cloudbreak.common.exception.BadRequestException;
 import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.common.mappable.ProviderParameterCalculator;
 import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
 import com.sequenceiq.cloudbreak.domain.SecurityGroup;
 import com.sequenceiq.cloudbreak.domain.Template;
 import com.sequenceiq.cloudbreak.domain.stack.instance.InstanceGroup;
 import com.sequenceiq.cloudbreak.domain.stack.instance.InstanceMetaData;
-import com.sequenceiq.cloudbreak.common.exception.BadRequestException;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import static java.util.Map.entry;
 
 @Component
 public class InstanceGroupV4RequestToInstanceGroupConverter extends AbstractConversionServiceAwareConverter<InstanceGroupV4Request, InstanceGroup> {
 
+    public static final int DEFAULT_UPDATE_DOMAIN_COUNT = 20;
```

Comment:
This was intentional. The Basic SKU Azure load balancer can only route traffic to instances it shares an availability set with. However, we were not previous grouping instances in the stack into availability sets. So this logic was added to group instances such that each instance group has its own availability set, so that we can add the load balancer to the appropriate availability.

Reply:
If that is the case, pls consult with @doktoric as we removed AS setting by default